tests/FiberTraces.py

```diff
@@ -56,10 +56,7 @@ def testFiberTraceFunctionFindingControl(self):
         
         """Test that we can set the parameters of the FiberTraceFunctionFindingControl"""
         interpolation = "POLYNOMIAL"
-        print "interpolation = ", type(interpolation),": <",interpolation,">"
-        print "ftffc.fiberTraceFunctionControl.interpolation = ", type(self.ftffc.fiberTraceFunctionControl.interpolation),": <",self.ftffc.fiberTraceFunctionControl.interpolation,">"
```

Comment:
Why is this commit separate from d398c78?

In general, please rebase to cleanup your tickets before sending for review.  E.g. all the printing cleanup (in python and C++) could/should be in a single changeset that is clearly identified and does nothing else.